core/ui/EditTemplate/type.tid

```diff
@@ -2,36 +2,113 @@ title: $:/core/ui/EditTemplate/type
+  emptyMessage="""<<cancel-delete-tiddler-actions "cancel">>"""
+>
+  <$action-sendmessage
+    $message="tm-remove-field"
+    $param="type"
+  />
+  <$action-deletetiddler
+    $filter="[<typeInputTiddler>] [<refreshTitle>] [<typeSelectionTiddler>]"
+  />
 </$list>
-</$linkcatcher>
-</div>
-</$reveal>
-</$set>
-</div>
-</div>
-</div>
+\end
+<$set name="refreshTitle" value=<<qualify "$:/temp/type-search/refresh">>>
+  <div class="tc-edit-type-selector-wrapper">
+    <em class="tc-edit tc-small-gap-right"><<lingo Type/Prompt>></em>
+    <div class="tc-type-selector-dropdown-wrapper">
+      <div class="tc-type-selector">
+        <$fieldmangler>
+          <$macrocall
+            $name="keyboard-driven-input"
+            tiddler=<<currentTiddler>>
+            storeTitle=<<typeInputTiddler>>
+            refreshTitle=<<refreshTitle>>
+            selectionStateTitle=<<typeSelectionTiddler>>
+            field="type"
+            tag="input"
+            default=""
+            placeholder={{$:/language/EditTemplate/Type/Placeholder}}
+            focusPopup=<<qualify "$:/state/popup/type-dropdown">>
+            class="tc-edit-typeeditor tc-edit-texteditor tc-popup-handle"
+            tabindex={{$:/config/EditTabIndex}}
+            focus={{{ [{$:/config/AutoFocus}match[type]then[true]] ~[[false]] }}}
+            cancelPopups="yes"
+            configTiddlerFilter="[[$:/core/ui/EditTemplate/type]]"
+            inputCancelActions=<<input-cancel-actions>>
+          />
+          <$button
+            popup=<<qualify "$:/state/popup/type-dropdown">>
+            class="tc-btn-invisible tc-btn-dropdown tc-small-gap"
+            tooltip={{$:/language/EditTemplate/Type/Dropdown/Hint}}
+            aria-label={{$:/language/EditTemplate/Type/Dropdown/Caption}}
+          >
+            {{$:/core/images/down-arrow}}
+          </$button>
+          <$button
+            message="tm-remove-field"
+            param="type"
+            class="tc-btn-invisible tc-btn-icon"
+            tooltip={{$:/language/EditTemplate/Type/Delete/Hint}}
+            aria-label={{$:/language/EditTemplate/Type/Delete/Caption}}
+          >
+            {{$:/core/images/delete-button}}
+            <$action-deletetiddler
+              $filter="[<typeInputTiddler>] [<storeTitle>] [<refreshTitle>] [<selectionStateTitle>]"
+            />
+          </$button>
+        </$fieldmangler>
+      </div>
+      <div class="tc-block-dropdown-wrapper">
+        <$set name="tv-show-missing-links" value="yes">
+          <$reveal
+            state=<<qualify "$:/state/popup/type-dropdown">>
+            type="nomatch"
+            text=""
+            default=""
+          >
+            <div class="tc-block-dropdown tc-edit-type-dropdown">
+              <$linkcatcher to="!!type">
+                <$list
+                  filter='[all[shadows+tiddlers]prefix[$:/language/Docs/Types/]each[group]sort[group-sort]]'
+                >
```

Comment:
I think, if there is only 1 line, it should stay as 1 line. It should not be formatted that way